corehq/apps/hqadmin/service_checks.py

```diff
@@ -53,7 +53,11 @@ def check_redis():
     if 'redis' in settings.CACHES:
         import redis
         rc = cache.caches['redis']
-        redis_api = redis.StrictRedis.from_url('%s' % rc._server)
+        if isinstance(rc._server, list):
+            redis_server = rc._server[0]
+        else:
+            redis_server = rc._server
+        redis_api = redis.StrictRedis.from_url('%s' % redis_server)
```

Comment:
nit: I know you didn't change this, but this is not the most obvious way to convert `redis_server` to a string.
```suggestion
        redis_api = redis.StrictRedis.from_url(str(redis_server))
```

Reply:
I think it is already a string, Im not sure why we use string formatting instead of just passing in the variable, but I will change that